Playbooks/CybleLogicApp/azuredeploy.json

```diff
@@ -0,0 +1,641 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+     "metadata": {
+        "title": "Cyble-Threat-Intel-Playbook",
+        "description": "The Cyble Threat Intel Playbook facilitates the retrieval of logs from the Cyble Alerts API into Microsoft Sentinel.",
+        "prerequisites": [
+            "A Cyble Alerts Token and Cyble Alerts Endpoint is required for authentication. Obtain your unique from [Cyble Vision Application](https://cyble.ai) for deployment."
+        ],
+        "postDeployment": [
+            "After deployment, open your Microsoft Sentinel Workspace to track latest incidents with the tag Cyble-Incidents"
+        ],
+        "prerequisitesDeployTemplateFile": "",
+        "entities": [],
+        "lastUpdateTime": "2024-03-04T19:42:00.000Z",
+        "support": {
+            "tier": "Developer",
+            "name": "Cyble",
+            "email": "support@cyble.com",
+            "link": "https://cyble.com"
+        },
+        "author": {
+            "name": "Cyble"
+        },
+        "releaseNotes": [
+            {
+                "version": "1.0.0",
+                "title": "Cyble-Threat-Intel",
+                "notes": [ "Initial version" ]
+            }
+        ]
+    },
+    "parameters": {
+        "workflowName": {
+            "defaultValue": "CybleThreatIntel",
+            "type": "String",
+            "metadata": {
+                "description": "Name of Cyble Connector Logic App"
+            }
+
+        },
+        "workspaceName": {
+            "defaultValue": "CybleLogAnalyticsWorkspace",
```

Comment:
Hello @cyble-dev, You have used workspace name as default for "CybleLogAnalyticsWorkspace"

Do You want user to use this Workspace as default for Playbook or the user can use his own. 
As mentioned in Description, I guess it would be fine if the user adds his own workspace where Cyble connector is setup

Reply:
Yes, the user can use their own. I will remove the dafault value if needed.